Quadriga/src/main/webapp/WEB-INF/views/public/publicWebsiteEditAbout.jsp

```diff
@@ -0,0 +1,14 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
```

Comment:
This is the jsp page for the about text settings page, right? This should be in the public folder but in the appropriate folder under the auth folder.